service/controller/aws/v14/resource_set.go

```diff
@@ -22,15 +22,16 @@ import (
 	"github.com/giantswarm/cluster-operator/pkg/label"
 	chartconfigservice "github.com/giantswarm/cluster-operator/pkg/v14/chartconfig"
 	configmapservice "github.com/giantswarm/cluster-operator/pkg/v14/configmap"
+	"github.com/giantswarm/cluster-operator/pkg/v14/kubeconfig"
 	"github.com/giantswarm/cluster-operator/pkg/v14/resource/certconfig"
 	"github.com/giantswarm/cluster-operator/pkg/v14/resource/chartoperator"
 	"github.com/giantswarm/cluster-operator/pkg/v14/resource/encryptionkey"
 	"github.com/giantswarm/cluster-operator/pkg/v14/resource/namespace"
 	"github.com/giantswarm/cluster-operator/pkg/v14/resource/tiller"
 	"github.com/giantswarm/cluster-operator/service/controller/aws/v14/key"
+	awskey "github.com/giantswarm/cluster-operator/service/controller/aws/v14/key"
```

Comment:
It would be useful if you could review your own code upfront. Line 31 and 32 import the same package. Also, there is no reason to alias the import.